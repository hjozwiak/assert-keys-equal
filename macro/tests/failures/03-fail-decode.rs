#[macro_use]
extern crate solana_misc_macro;
use solana_program::pubkey::Pubkey;
struct Tester {
    key: Pubkey,
}

fn main() {
    let test = Tester {
        key: Pubkey::new_unique(),
    };
    assert_pubkeys_equal!(test.key, "ishouldfailtodecodethisstringstuff");
}
