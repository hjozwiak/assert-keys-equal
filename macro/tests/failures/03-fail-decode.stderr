error: failed to decode base58 string
  --> $DIR/03-fail-decode.rs:12:37
   |
12 |     assert_pubkeys_equal!(test.key, "ishouldfailtodecodethisstringstuff");
   |                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
